components/security/secure_tools.rst

```diff
@@ -1,47 +1,39 @@
-Securely Generating Random Numbers
+Securely Generating Random Strings 
 ==================================
 
 The Symfony Security component comes with a collection of nice utilities
 related to security. These utilities are used by Symfony, but you should
 also use them if you want to solve the problem they address.
 
-Generating a Secure random Number
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Generating a Secure random
```

Comment:
I've added documentation for random_int that should therefore address both of your concerns.